.github/workflows/ccruntime-on-push.yaml

```diff
@@ -0,0 +1,17 @@
+name: ccruntime e2e test on-push
```

Comment:
Nit: not use `on-push` on the workflow name, file and jobs as this workflow is triggered on pull request.

Reply:
Yes, I will rename ones relevant to this comment. Thanks for the comment.